lntest/itest/assertions.go

```diff
@@ -623,6 +623,121 @@ func txStr(chanPoint *lnrpc.ChannelPoint) string {
 	return cp.String()
 }
 
+// waitForChannelUpdate waits for a node to receive the expected channel
+// updates.
+func waitForChannelUpdate(t *harnessTest, subscription graphSubscription,
+	expUpdates []expectedChanUpdate) {
+
+	// Create an array indicating which expected channel updates we have
+	// received.
+	found := make([]bool, len(expUpdates))
+out:
+	for {
+		select {
+		case graphUpdate := <-subscription.updateChan:
+			for _, update := range graphUpdate.ChannelUpdates {
+				require.NotZerof(
+					t.t, len(expUpdates),
```

Comment:
We check the number of arguments passed to the function here. But the error message suggests something else should be checked here?